### DISCUSSION TITLE
Is it possible to support folder import? I used Qunhui Photos before, but now I want to migrate to immich

### DISCUSSION THREAD

**Discussion by ljlzzzzz** [immich-app/immich#7571] [General]
Is it possible to support folder import? I used Qunhui Photos before, but now I want to migrate to immich

**Comment by mmomjian** [comment]
You can upload a folder using the Immich CLI tool or Immich-go.

**Reply by ljlzzzzz** [reply]
thank you, i will try later

**Reply by ljlzzzzz** [reply]
![image](https://github.com/immich-app/immich/assets/42434249/d2ab5f7d-4880-4b44-991e-fdb6b30bc447)
 hi, I am using this method, but it keeps prompting the error 'no such file or directory'. I tried an absolute path but it didn't work. What is the default path for this method

**Comment by bo0tzz** [comment]
Or if you don't want to upload, you can include an external library: https://immich.app/docs/features/libraries

**Reply by ljlzzzzz** [reply]
thank you very mush
